### PR TITLE
Initial RPC Framework

### PR DESCRIPTION
**Pull request by ShivanshVij**

### CODE REVIEW

File: protoc-gen-frisbee/pkg/generator/client.go [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+	frisbeeClient = "*frisbee.Client"
+)
+
+func writeClientHandlers(f File, services protoreflect.ServiceDescriptors) {
+	operationIndex := 0
+	for i := 0; i < services.Len(); i++ {
+		service := services.Get(i)
+		for j := 0; j < service.Methods().Len(); j++ {
+			method := service.Methods().Get(j)
+			f.P(tab, "table[", operationIndex+operationOffset, "] = ", handlerSignature, " {")
```
Comment (Jspsun) [inline protoc-gen-frisbee/pkg/generator/client.go:35]: dear lord we should probably generate these with template files. I swear I wrote code to do this

File: protoc-gen-frisbee/pkg/generator/server.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+		builder.WriteString(comma)
+		builder.WriteString(space)
+	}
+	serverFields := builder.String()
+	serverFields = serverFields[:len(serverFields)-2]
+	f.P("func NewServer(", serverFields, ", listenAddr string, tlsConfig *tls.Config, logger *zerolog.Logger) (*Server, error) {")
+	f.P(tab, "table := make(frisbee.HandlerTable)")
+	writeServerHandlers(f, services)
+
+	f.P(tab, "var s *frisbee.Server")
```
Comment (Jspsun) [inline protoc-gen-frisbee/pkg/generator/server.go:85]: for a lot of fixed text, we should probs move these into template files

**Review by Jspsun — DISMISSED** [review]
LGTM. I think there are a lot of things that will be better as template files but this definitely works. Some other thoughts: does this work against mal-formed protos. Does the protoc-gen wrapper handle that instead?

If we want to punt on the above, happy to let this merge in for velocity's sake and we can address tech debt later and cut tickets for it

**Comment by ShivanshVij** [comment]
Created an issue to address tech debt later: https://github.com/loopholelabs/frisbee/issues/79

**Pull request merged**